spec/models/service_template_transformation_plan_request_spec.rb

```diff
@@ -31,8 +31,9 @@
 
   describe '#validate_conversion_hosts' do
     context 'no conversion host exists in EMS' do
-      let(:dst_ems) { FactoryBot.create(:ext_management_system) }
-      let(:src_cluster) { FactoryBot.create(:ems_cluster) }
+      let(:src_ems) { FactoryBot.create(:ems_vmware) }
+      let(:dst_ems) { FactoryBot.create(:ems_openstack) }
+      let(:src_cluster) { FactoryBot.create(:ems_cluster, :ext_management_system => src_ems) }
```

Comment:
Which is exactly what you're doing here

Reply:
@agrare did you state the observation or are you asking a question?  The addition was needed to deal with the added validation in in TransformationMappingItem

Reply:
Nope just stating that this is all that needs to be done to work w/o traits